core/templates/app.constants.ts

```diff
@@ -71,6 +71,8 @@ export const AppConstants = {
     INCORRECT_SOLUTION: 'The current solution does not lead to another card.',
     UNRESOLVED_ANSWER:
       'There is an answer among the top 10 which has no explicit feedback.',
+    INVALID_REDIRECTION: 'Learner should not be directed back by more' +
+      ' than MAX_CARD_COUNT_FOR_VALID_REDIRECTION cards in the lesson.'
```

Comment:
This is wrong. The value "3" should be interpolated dynamically from the `MAX_CARD_COUNT_FOR_VALID_REDIRECTION` constant.

Reply:
Fixed.